### DISCUSSION TITLE
Checkbox in cards don't can't be marked

### DISCUSSION THREAD

**Discussion by Sem-the-dev** [community/community#10726] [Projects and Issues]
The checkbox in the cards can't be marked as done

**Comment by iramosp** [comment]
I have this issue in Projects Beta: checkboxes in cards (ie items that are "draft issues") show up like in the old project boards but they can't be clicked. To mark them as done, I have to edit the text directly to add the `x`.

**Reply by BirgitPohl** [reply]
Indeed! Adding the same story from my side.
